src/android/FileTransfer.java

```diff
@@ -977,7 +977,11 @@ public void run() {
                         context.sendPluginResult(new PluginResult(PluginResult.Status.ERROR, error));
                         context.aborted = true;
                         if (context.connection != null) {
-                            context.connection.disconnect();
+                            try{
+                                   context.connection.disconnect();
+                            } catch (Exception e) {                              
+                                   Log.e(LOG_TAG, "Catch workaround for fatal exception", e);
```

Comment:
@virix, could you please also prefix the error message with `"CB-8431"` to simplify a potential debugging?